lib/tbot/config/output_database.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gravitational/trace"
+	"golang.org/x/exp/slices"
+	"gopkg.in/yaml.v3"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/tbot/bot"
+	"github.com/gravitational/teleport/lib/tbot/identity"
+)
+
+const DatabaseOutputType = "database"
+
+// DatabaseFormat specifies if any special behavior should be invoked when
+// producing artifacts. This allows for databases/clients that require unique
+// formats or paths to be used.
+type DatabaseFormat string
+
+var (
```

Comment:
```suggestion
const (
```

I think this should work for all of these, though the `databaseFormats` array may still need to be in a var.

Reply:
Yup - consts seem like a better idea here, thanks!

Reply:
06dcdd5e5c